pdr_backend/lake_info/test/test_lake_info.py

```diff
@@ -0,0 +1,210 @@
+import io
+from unittest.mock import MagicMock
+
+import polars as pl
+from enforce_typing import enforce_types
+
+from pdr_backend.lake_info.overview import TableViewsOverview, ValidationOverview
+
+
+csv_string = """
```

Comment:
I'd put csv_string into a separate file, like pdr_backend/lake_info/test/resources.py

It will make test_lake_info far more readable & maintainable.